### PR TITLE
fix telemetry when auth is off

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -71,6 +71,6 @@ wait-for-url() {
 }
 
 # Dashboard service
-(cd /dashboard; wait-for-url localhost:8088/v1/version && yarn start:docker) &
+(cd /dashboard; wait-for-url localhost:8088/v1/version && NEXT_PUBLIC_UID=$(cat /proc/sys/kernel/random/uuid) yarn start:docker) &
```
Comment (dfellis) [inline docker-entrypoint.sh:74]: Oh wow, I had no idea The Linux kernel itself sprouted a UUID generator. That's cool :)

File: dashboard/package.json [hunk 2]
```diff
@@ -43,7 +43,6 @@
     "typescript": "4.9.5",
     "unique-names-generator": "^4.7.1",
     "universal-cookie": "^4.0.4",
-    "uuid": "^9.0.0",
```
Comment (dfellis) [inline dashboard/package.json:46]: Did you re-run `yarn` after this? I'm surprised the `yarn.lock` file was not touched.

Comment (depombo) [reply dashboard/package.json:46]: I did and it didn't regenerate it. I was also surprised

File: docker-entrypoint.sh (hunk 1 above)
Comment (depombo) [reply docker-entrypoint.sh:74]: Yes, I also learned the today!

**Review by dfellis — APPROVED** [review]

**Review by aguillenv — APPROVED** [review]

**Pull request merged**